src/azure-cli/azure/cli/command_modules/security/custom.py

```diff
@@ -113,6 +116,19 @@ def get_security_setting(client, resource_name):
     return client.get(resource_name)
 
 
+def update_security_setting(client, resource_name, is_enabled):
```

Comment:
According to the swagger definition. You should define an argument --setting-name with enum values instead of reuse the argument resource_name.

Reply:
Good idea, will change